src/domains/leaderboard/leaderboard.module.ts

```diff
@@ -1,16 +1,9 @@
 import { Module } from '@nestjs/common';
-import { LeaderboardTokens } from './interfaces';
 import { LeaderboardController } from './leaderboard.controller';
-import { LeaderboardService } from './service/leaderboard.service';
+import { PrismaModule } from '../../shared/prisma/prisma.module';
 
 @Module({
+  imports: [PrismaModule],
```

Comment:
I don't think you need to import prismaModule here because it's a global module in app.module.ts.

Reply:
tx correct and commited ! @schwepps